apps/server/src/services/TimerService.ts

```diff
@@ -316,6 +321,15 @@ export class TimerService {
   _onFinish() {
     eventStore.set('timer', this.timer);
     integrationService.dispatch(TimerLifeCycle.onFinish);
+    if (this.timer.endAction === EndAction.Stop) {
+      PlaybackService.stop();
+    } else if (this.timer.endAction === EndAction.LoadNext) {
+      setTimeout(() => {
+        PlaybackService.loadNext();
+      }, 0);
```

Comment:
It is nice to document when you need to add a weird workaround.
Our future selves will not remember what was this all about